stylesheets/pages/quiz.css

```diff
@@ -0,0 +1,53 @@
+@import url(../base.css);
+/* Quiz Page  */
+
+.quiz-question-wrapper{
+    width: 90%;
+    max-width: 500px;
+    border-radius: var(--space-xs);
+    gap: var(--space-xlg);
+    transition: background-color 0s;
+}
+
+.quiz-question-wrapper >*{
+    border-radius: var(--space-xs) ;
+    background: var(--tr-background-lighter);
+}
+
+.ques-info .timer-wrapper{
+    position: absolute;
+    left: 50%;
+}
+
+.question-label{
+    border-radius: var(--space-xs);
+    top: -20px;
+    padding: 5px 20px;
+    color: var(--tr-background);
+    /* background: var(--tr-primary-lighter); */
+}
+
+.ques-options .d-flex{
+    gap: var(--space-md);
+}
+
+.ques-options .tr-btn{
+    flex: 1 1 10rem;
+}
+
+.ques-options *{
+    flex: 1;
+}
+
+.time-indicator-bar{
+    height: 5px;
+    width: 100%;
+    background: #00ff00;
```

Comment:
You can use variable for this color

Reply:
actually that color is needed there because I'm changing the hex values using JS and operating on hex value is relatively easier, so I can't use CSS Variables for that